### ISSUE TITLE
[4.0] [a11y] Select User in Created by field is not focusable

### ISSUE THREAD

**Issue by Harshil99** [joomla/joomla-cms#24323]
### Steps to reproduce the issue
Go to Article/New/Publishing
Tab through the fields.

### Expected result
Select user (icon beside the field) should be focusable.

### Actual result
Field is focusable (Not needed as nothing can be done there if only selection is to be done that is also from another container) not Select user icon beside it.

### System information (as much as possible)
Joomla! 4.0.0-alpha8-dev
Google Chrome Version 72.0.3626.121 (Official Build) (64-bit)
Ubuntu 18.04.2 LTS

**Comment by brianteeman** [comment]
Confirmed it is not selectable

**Comment by Quy** [comment]
Please test PR #24330<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24323">issues.joomla.org/tracker/joomla-cms/24323</a>.</sub>
